applications/crossbar/src/modules/provisioner_v5.erl

```diff
@@ -349,14 +349,37 @@ settings_keys(Assoc, KeyKind, JObj) ->
         LineKey -> kz_json:set_value(<<"account">>, LineKey, Keys)
     end.
 
+-spec get_label(binary(), binary()) -> binary().
+get_label(AccountId, DocId) ->
+    AccountDb = kz_util:format_account_id(AccountId, 'encoded'),
+    Doc = kz_datamgr:open_cache_doc(AccountDb, DocId),
+
+    get_label(Doc).
+
+-spec get_label(kz_json:object()) -> binary().
+get_label(Doc) ->
+    case {kz_json:get_value(<<"first_name">>, Doc), kz_json:get_value(<<"last_name">>, Doc)} of
+        {'undefined', 'undefined'} ->
+            kz_json:get_value(<<"name">>, Doc);
+
+        {First, 'undefined'} ->
+            First;
+
+        {'undefined', _Last} ->
+            kz_json:get_value(<<"name">>, Doc);
```

Comment:
Space because it looks good :)

In reality I think it makes it easier to read, but I don't have a strong opinion either way.

As for just returning last, I was thinking it through and that just seemed like odd behavior.  I suppose that I could be convinced differently.